src/toInteger32.js

```diff
@@ -0,0 +1,21 @@
+import { curryN } from 'ramda';
+/**
+ * Converts double-precision 64-bit binary format IEEE 754 to signed 32 bit integer number.
+ *
+ * @func toInteger32
+ * @memberOf RA
+ * @since {@link https://char0n.github.io/ramda-adjunct/1.12.0|v1.12.0}
```

Comment:
```suggestion
 * @since {@link https://char0n.github.io/ramda-adjunct/2.28.0|v2.28.0}
```

Reply:
Thanks, I was unsure how to populate this one